src/index.html

```diff
@@ -10,13 +10,105 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        class="logo"
+        href="#home"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo Company Logo"
+        />
+      </a>
```

Comment:
Add empty lines between multiline sibling blocks of HTML. There should be an empty line before the <header> tag to separate it from the <body> tag.